### ISSUE TITLE
Support Internationalization (other languages)

### ISSUE THREAD

**Issue by alganzory** [alganzory/HaramBlur#16]

**Comment by anonfaded** [comment]
Working on this feature brother, can you assign this issue to me :)

**Comment by alganzory** [comment]
Thanks @anonfaded

**Comment by anonfaded** [comment]
Brother i have added arabic language to the HaramBlur extension but there are some issues which i can't figure out so need guidance.

 **I did two things:**
1. Added `IDs` in the tags in `popup.html` file.
2. Added `translations.js` file which contains the translations. 

But the issue is with the `tooltip`, the `blur` and the `strictness` `%` value which isn't showing on both english and arabic version which you can see in the attached screenshot below.
![Screenshot from 2023-12-16 18-14-03](https://github.com/alganzory/HaramBlur/assets/124708903/a3e04fb0-c434-4c6b-969b-147cc5877b67)

The buttons of grayscale, image etc all are working fine and it blurs the images but it doesn't show the percentage.

Also when i load the extension it loads like you saw above but when i upload the unzipped extension which we get from `npm run release` command, then the arabic translation when picked from the dropdown doesn't work and also the tooltip and `%` is visible as it is now in the online extension.

Sharing a fullscreen screenshot below and waiting for your suggestion on what to do now, should i open the pull request if you can handle the rest? or any guidance on how to fix this issue.
![Screenshot from 2023-12-16 20-22-33](https://github.com/alganzory/HaramBlur/assets/124708903/05ec9bff-3bbd-45c9-9c50-b8f1791f668b)



## Check the code here: 
https://github.com/anonfaded/HaramBlur

**Comment by alganzory** [comment]
Thank you for your efforts, May Allah bless you

I think it's best if you open a PR first and then I'll look at the changes and see what could be causing the bugs. For one, I think the build problem you're getting is because you haven't added the new files to the release command in `package.json`. 
But again, please open a PR so I can look into it further

Jazak Allahu Khairan

**Comment by anonfaded** [comment]
yes please review the code, and I'm here for any further steps that i may have missed.
If everything goes fine and we find the solution then i will recheck the translations and will correct them after i verify that from arabic speaking friend.
